### ISSUE TITLE
Create diagram for the site to show how attestations are generated and common methods to distribute them.

### ISSUE THREAD

**Issue by mlieberman85** [slsa-framework/slsa#477]
Related to: https://github.com/slsa-framework/slsa/issues/475

We already highlight that JSON lines files and OCI are ways to store and distribute SLSA attestations, but people are a bit confused about how to use them in common patterns.

**Comment by sethmlarson** [comment]
Hello! I was confused about how SLSA/in-toto/DSSE all mapped together and looking at the SVG on [this page](https://slsa.dev/attestation-model) was a little confusing, especially knowing which pieces correspond to which files/specifications I know about. I took an attempt at creating my own diagram in draw.io:

![slsa](https://user-images.githubusercontent.com/18519037/189401424-6e05fe32-172d-4564-82ad-4b1f0ce96834.svg)

Another benefit of this SVG is it's more vertical rather than horizontal which is more universal in terms of directional hierarchies (left-to-right/right-to-left are confusing to at least one part of the world) and is more mobile friendly!

I've also included the .drawio file so it can be modified:

[slsa.drawio.txt](https://github.com/slsa-framework/slsa/files/9537326/slsa.drawio.txt)

Let me know if there is interest in using a diagram like this or if changes need to be made.
